tools/releaseBuild/azureDevOps/releasePipeline.yml

```diff
@@ -139,24 +147,83 @@ stages:
   dependsOn: []
   displayName: Manual Validation
   jobs:
-  - deployment: ValidateWinPkg
-    displayName: Validate Windows Packages
-    pool: server
-    environment: PSReleaseWinPkgValidate
+  - template: templates/release/approvalJob.yml
+    parameters:
+      displayName: Validate Windows Packages
+      jobName: ValidateWinPkg
+      instructions: |
+        Validate zip and msipackages on Windows Server 2012 R2
 
-  - deployment: ValidateLinuxPkg
-    displayName: Validate Linux Packages
-    pool: server
-    environment: PSReleaseLinuxPkgValidate
+  - template: templates/release/approvalJob.yml
+    parameters:
+      displayName: Validate OSX Packages
+      jobName: ValidateOsxPkg
+      instructions: |
+        Validate tar.gz package on osx-arm64
 
 - stage: ReleaseAutomation
   displayName: Release Automation
   dependsOn: []
   jobs:
-  - deployment: ReleaseAutomationKickOff
-    displayName: Kick off release ReleaseAutomation
-    pool: server
-    environment: PSReleaseAutomationKickOff
+  - job: KickOffRA
+    displayName: Kickoff Release Automation
+    steps:
+    - checkout: Internal-PowerShellTeam-Tools
+    - task: DownloadPipelineArtifact@2
+      inputs:
+        source: specific
+        project: PowerShellCore
+        pipeline: '696'
+        preferTriggeringPipeline: true
+        runVersion: latestFromBranch
+        runBranch: '$(Build.SourceBranch)'
+        artifact: metadata
+        path: '$(Pipeline.Workspace)/releasePipeline/metadata'
+
+    - pwsh: |
+        Import-Module $(Agent.BuildDirectory)\Internal-PowerShellTeam-Tools\ReleaseTools\AzDO -Force
+        Set-AzDoAuthToken -Token $(RELEASEAUTOMATIONPAT)
+        Set-AzDoProjectInfo -ProjectOwner PowerShell-Rel -ProjectName Release-Automation
+        $packageBuildID = $(resources.pipeline.releasePipeline.runID)
+        $metadata = Get-Content -Raw -Path '$(Pipeline.Workspace)/releasePipeline/metadata/release.json' | ConvertFrom-Json
+        $buildInvocationInfo = Start-AzDOBuild -BuildDefinitionId 10 -BuildArguments @{ POWERSHELL_PACKAGE_BUILD_BUILDID = $packageBuildID } -Tag $metadata.ReleaseVersion, 'InProgress'
+        Write-Verbose -Verbose "Kicked off release automation: $($buildInvocationInfo.WebUrl)"
+        $status = $buildInvocationInfo | Wait-AzDOBuildStatus -Status Completed -TimeoutInMinutes 300
+        if ($status.result -ne 'Succeeded') {
+          Write-Verbose "There are errors in release automation tests. Please triage failures."
```

Comment:
for the moment, you'll probably always get this message

Reply:
yup